README.md

```diff
@@ -1 +1,66 @@
 # snare
+
+`snare` is a simple GitHub webhooks runner. When a request comes in, it runs an
+arbitrary program for that repository informing it of the event -- that program
+can then perform whatever actions it wants.
+
+
+## Basic usage
+
+`snare` requires three command-line arguments to be specified:
+
+```
+Usage: snare -p <port> -r <repo scripts dir> -s <secret>
```

Comment:
The secret is going to appear in the process table if it is an argument to the command. Better to read from a file?

Reply:
I think `<repo-scripts-dir>` is easier to read (with hyphens), but I don't feel strongly.

Reply:
> The secret is going to appear in the process table if it is an argument to the command. Better to read from a file?

Good point.